src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -215,6 +228,35 @@ export const LocatorEditDialog: React.FC<Props> = ({
     updateRef(OnbrdStep.EditLocator, undefined, isOkButtonDisabled ? undefined : handleCreateCustomLocator);
   };
 
+  const navigateToOriginalLocator = async (event: MouseEvent<HTMLSpanElement>) => {
+    event.stopPropagation();
+    const _locator = form.getFieldValue("locator");
+    form.resetFields();
+    setIsModalOpen(false);
```

Comment:
I suggest to extract these two operations into a new function and reuse it other places of the component

Reply:
fixed